packages/components/src/FormField/hooks/useToolbar.ts

```diff
@@ -0,0 +1,25 @@
+import { FormFieldProps } from "../FormFieldTypes";
+
+interface UseToolBarProps {
+  focused: boolean;
+  toolbar: React.ReactNode;
+  toolbarVisibility: FormFieldProps["toolbarVisibility"];
+}
+
+interface UseToolbar {
+  isToolbarVisible: boolean;
+  animationInitial: boolean | { opacity: number; height: number };
+}
+
+export function useToolbar(props: UseToolBarProps): UseToolbar {
+  const isToolbarVisible =
+    props.toolbar !== undefined &&
+    (props.toolbarVisibility === "always" || props.focused);
+  const animationInitial =
+    props.toolbarVisibility === "always" ? false : { opacity: 0, height: 0 };
```

Comment:
would passing in `initial={true || false}` in the animated presence do what this is trying to do?

Reply:
I'm not sure I follow. `true || false`  that's just always going to be `true` no?

also, are you asking to swap to `AnimatedPresence`? because this is using `AnimatePresence` from Framer, not the Atlantis component

on that note, those names are incredibly similar

Reply:
oh no, animate presence. just saying there's a prop to take out the initial animations on any motion children haha

Reply:
ah gotcha. I'll take a look, this is my first time using the API so definitely not sure what the best way to manage some of this stuff is